src/main/java/org/terasology/ligthandshadow/componentsystem/components/HasFlagComponent.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.ligthandshadow.componentsystem.components;
+
+import org.terasology.entitySystem.Component;
+
+public class HasFlagComponent implements Component {
```

Comment:
can you add javadoc for this component, explaining which entities can have this component